src/SeleniumLibrary/keywords/element.py

```diff
@@ -164,21 +164,47 @@ def page_should_contain_element(self, locator, message='', loglevel='INFO'):
         self.assert_page_contains(locator, message=message, loglevel=loglevel)
 
     @keyword
+    def get_matching_locator_count(self, locator):
```

Comment:
Should have `return_str` argument similarly as `get_matching_xpath_count` that this keyword deprecates.

Reply:
I agree that for backwards compatible reasons it should be there. But I don't like that argument, because `get_matching_xpath_count` by default returns a string and it would be better keyword functionality if this the keyword would return a integer. If string is really needed, one can use `Convert To String` keyword.

Reply:
Oh, I thought the default was int and there was some strange reason to sometimes return str! I totally agree that int is a better return type and that no configuration is needed. Just need to document the change. At least the old keyword should have a note that the new one has different return value type.